runtime_util/runtime_util.h

```diff
@@ -24,9 +24,12 @@
 
 #define DEVICES_LENGTH 64  // The largest device type number + 1.
 
-#define NUM_DESC_FIELDS 5  // Number of fields in the robot description
+#define NUM_DESC_FIELDS 6  // Number of fields in the robot description
 
-#define NUM_GAMEPAD_BUTTONS 17  // Number of gamepad buttons
+#define LOG_KEY_LENGTH 64  // Max length of a key for custom Robot.log data
```

Comment:
Can we name this `MAX_LOG_KEY_LENGTH`

Reply:
that gets pretty long, i think its fine as is